docs/source/conventions/example.rst

```diff
@@ -29,56 +29,26 @@ sequence derived from another source.
 In this example, we will use a :ref:`SequenceLocation`, which is
 composed of a sequence identifier and a :ref:`SequenceInterval`.
 
-In VRS, all identifiers are a |CURIE|.  Therefore, NC_000013.11 MUST be
-written as the string ``refseq:NC_000019.10`` to make explicit that
-this sequence is from `RefSeq
-<https://www.ncbi.nlm.nih.gov/refseq/>`__ .  VRS does not restrict
-which data sources may be used, but does recommend using prefixes from
-`identifiers.org <http://identifiers.org>`_.
```

Comment:
Similar as above. I just removed the entire section. Did we want to say anything in particular about RefGet accessions being required now?

Reply:
again, good call and appropriate to remove. It might be useful to add a few words about RefGet here. Perhaps revise above paragraph to discuss :ref:`SequenceReference` instead of "computed identifier" and remove the reference to :ref:`SequenceInterval`.

Reply:
Ope, good catch. Missed deleting interval reference.